open-craft/ansible-playbooks
# [FAL-1765] Add isolated MySQL server for Jira

This pull request is about adding a new isolated MySQL server for Jira.

Related pull requests:

- https://github.com/open-craft/ansible-secrets/pull/259
- https://github.com/open-craft/terraform-secure/pull/3
- [opencraft/documentation/private#539](https://gitlab.com/opencraft/documentation/private/-/merge_requests/539)

**Jira tickets**

- [FAL-1765](https://tasks.opencraft.com/browse/FAL-1765)

**Testing instructions**

Check that Jira is working properly.
Run `tarsnap --key-file <KEY_FILE> --list-archives` and check that archives are present.
Read the changes.

**Reviewers**

TBA
